utils/vpr_place_view.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+"""
```

Comment:
Would be nice to add an output image to the pull request.

Reply:
Added.